docs/getting-started.md

```diff
@@ -314,6 +314,17 @@ arguments is available via:
 tools/devtool --help
 ```
 
+### Alternative: Building Firecracker without devtool
+
+It is possible to build Firecracker without invoking `devtool` by instead running
+`cargo build`. However, binaries generated like this should not be used in production 
+and are considered experimental. 
+
+If you are trying to build Firecracker without `devtool` on an ARM machine, be advised 
+that our `.cargo/config` files specifies the default target to be `x86_64-unknown-linux-musl`,
+meaning you will by default be cross-compiling. Should this not be desired, you
+have to manually pass the `--target aarch64-unknown-linux-musl` flag to `cargo build`.
```

Comment:
Nit: It could be slightly re-worded to be a bit more concise and a bit less prescriptive.
```suggestion
By default Firecracker builds for`x86_64-unknown-linux-musl` target (set in `.cargo/config`). To build for another target you will need to manually pass the `--target` flag to `cargo build` (e.g. `cargo build --target aarch64-unknown-linux-musl`).
```

Reply:
Mh, the crux to me here is that you have to specify the `--target` flag even though you want to compile for ARM on ARM (e.g. to _not_ cross-compile, you have to pass `--target`). Let me see if I can get that conveyed a bit more precisely